csaf_2.1/prose/edit/src/guidance-on-size.md

```diff
@@ -228,14 +228,14 @@ A string SHOULD NOT have a length greater than:
   * `/document/references[]/summary`
   * `/document/tracking/revision_history[]/summary`
   * `/product_tree/branches[]/product/product_identification_helper/cpe`
-  * `/product_tree/branches[]/product/product_identification_helper/purl`
+  * `/product_tree/branches[]/product/product_identification_helper/purls[]`
```

Comment:
`/product_tree/*/product/product_identification_helper/purl` (in all possible instances need also to be added to the array section.